src/olympia/api/utils.py

```diff
@@ -16,11 +17,19 @@ def is_gate_active(request, name):
     return name in gates
 
 
+class APIChoiceEntry(ChoiceEntry):
+    @property
+    def api_value(self):
+        return str(self.value).lower() if self.value else self.value
+
+
 class APIChoices(Choices):
     """Like a regular extended_choices.Choices class, with an extra api_choices
     property that exposes constants in lower-case, meant to be used as choices
     in an API."""
 
+    ChoiceEntryClass = APIChoiceEntry
+
     @property
     def api_choices(self):
         return tuple((entry[1], entry[0].lower()) for entry in self.entries)
```

Comment:
maybe ... but I don't want to expand the scope of this patch to add tests to make sure it works as expected everywhere it's used (and the `api_choices` in the subclass isn't consistent as we output `None` rather than `none` that would need investigating...)